src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -106,41 +110,31 @@ const NavHeader = (props) => {
               </View>
             </>
           )}
-          <View
-            style={{
-              flexDirection: 'row',
-              alignItems: 'flex-start',
-              zIndex: 2,
-              position: 'absolute'
-            }}>
-            <TouchableOpacity onPress={() => backScreen()} style={styles.backbutton}>
-              <MemoIcArrowBackCircle width={normalize(32)} height={normalize(32)} />
-            </TouchableOpacity>
-
-            <Animated.View
+          <View style={styles.headerBarContainer(insets.top)}>
+            <View
               style={{
-                opacity: opacityImage
+                flexDirection: 'row',
+                alignItems: 'center'
               }}>
-              <TopicDomainHeader {...props} />
-            </Animated.View>
-          </View>
-          <View
-            style={[
-              styles.containerAction,
-              {
-                paddingRight: dimen.normalizeDimen(20),
-                zIndex: 2,
-                position: 'absolute',
-                right: 0
-              }
-            ]}>
-            {!isFollow && isHeaderHide ? (
-              <ButtonFollow handleSetFollow={onFollowButtonPress} />
-            ) : (
-              <TouchableOpacity onPress={onShareCommunity} style={styles.shareIconStyle}>
-                <ShareIconCircle color="black" width={32} height={32} />
+              <TouchableOpacity onPress={() => backScreen()} style={styles.backButton}>
+                <MemoIcArrowBackCircle width={normalize(32)} height={normalize(32)} />
               </TouchableOpacity>
-            )}
+              <Animated.View
+                style={{
+                  opacity: opacityImage
+                }}>
+                <TopicDomainHeader {...props} />
+              </Animated.View>
+            </View>
+            <View>
+              {!isFollow && isHeaderHide ? (
+                <ButtonFollow handleSetFollow={onFollowButtonPress} />
+              ) : (
+                <TouchableOpacity onPress={onShareCommunity}>
+                  <ShareIconCircle color="black" width={32} height={32} />
+                </TouchableOpacity>
+              )}
+            </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like you've removed some absolute positioning styles from the previous version of the code. If these changes are intentional and tested across different screen sizes and devices, then it looks good. However, if not, please ensure that removing these styles doesn't break the layout on certain devices or screen sizes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->